parallax/task.py

```diff
@@ -126,7 +128,7 @@ def start(self, nodenum, iomap, writer, askpass_socket=None):
                     close_fds=False, preexec_fn=os.setsid, env=environ)
         else:
             self.proc = Popen(self.cmd, stdin=PIPE, stdout=PIPE, stderr=PIPE,
-                    close_fds=False, start_new_session=True, env=environ)
+                    close_fds=False, start_new_session=True, env=environ, shell=self.is_local)
```

Comment:
Looks like parallax.call() is bound to "shell", then it brings the known limitation to interact with `crm configure`, just in case, from my experience .